frontend/app/views/fragments/form/feedback.scala.html

```diff
@@ -61,7 +61,9 @@ <h2 class="fieldset__headline">About you</h2>
                    class="input-text"
                    type="text"
                    required
-                   aria-required="true"/>
+                   aria-required="true"
+                   value="@name.getOrElse("")"
```

Comment:
Alternative for `name.getOrElse("")` is `name.mkString`, but your taste may differ!